src/tiles/grid/account-tile.ts

```diff
@@ -1,123 +1,74 @@
+import { css, html, LitElement, TemplateResult } from 'lit';
 import { customElement, property } from 'lit/decorators.js';
 import type { TileModel } from '../../models';
 
+import { baseTileStyles } from './styles/tile-grid-shared-styles';
+import '../image-block';
 import './tile-stats';
 
 @customElement('account-tile')
 export class AccountTile extends LitElement {
   @property({ type: Object }) model?: TileModel;
 
+  @property({ type: String }) baseImageUrl?: string;
+
   render() {
     return html`
-      <div class="account-tile-main">
-        <div id="title">
-          <h1 class="truncated">${this.model?.identifier}</h1>
-        </div>
-
-        <div class="account-info">
-          <div id="avatar-info">
-            <img
-              id="avatar"
-              alt="patron-avatar"
-              src="https://archive.org/services/img/${this.model?.identifier}"
-            />
+      <div class="container">
+        <div class="tile-details">
+          <div class="item-info">
+            ${this.getAvatar} ${this.getTitle} ${this.getArchivist}
           </div>
-
-          <span id="archivist-since">
-            Archivist since ${this.model?.dateAdded?.getFullYear()}
-          </span>
+          ${this.getTileStats}
         </div>
-
-        <tile-stats
-          .mediatype=${this.model?.mediatype}
-          .itemCount=${this.model?.itemCount}
-          .favCount=${this.model?.favCount}
-          .commentCount=${this.model?.commentCount}
-        >
-        </tile-stats>
       </div>
     `;
   }
 
-  static get styles() {
-    return css`
-      h1 {
-        color: black;
-        font-size: 16px;
-        margin: 0;
-      }
-
-      span {
-        font-size: 14px;
-        color: #2c2c2c;
-        margin: 0px;
-      }
-
-      .account-tile-main {
-        background-color: #fcf5e6;
-        border: 1px #2c2c2c;
-        border-radius: 4px;
-        box-shadow: 1px 1px 2px 0px;
-        height: 100%;
-        display: flex;
-        flex-direction: column;
-        text-align: center;
-        width: 100%;
-      }
-
-      .account-info {
-        flex-grow: 1;
-      }
-
-      #title {
-        padding: 5px 5px 0px 5px;
-        flex-shrink: 0;
-        height: 40px;
-      }
-
-      .account-tile-main:hover > #title > .truncated {
-        text-decoration: underline;
-      }
-
-      /** this is a workaround for Safari 15 where the hover effects are not working */
-      #title:hover > .truncated {
-        text-decoration: underline;
-      }
+  private get getAvatar(): TemplateResult {
+    return html`
+      <image-block
+        .model=${this.model}
+        .baseImageUrl=${this.baseImageUrl}
+        .viewSize=${'grid'}
+      >
+      </image-block>
+    `;
+  }
 
-      #avatar-info {
-        margin-top: 5px;
-        margin-bottom: 5px;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-      }
+  private get getTitle() {
```

Comment:
getter function for different blocks

Reply:
I like that these have been pulled outside of the render method. Could we name the getters more like `avatarTemplate`, `titleTemplate` etc, to stay consistent with how some of our other components name template getters? I think that more clearly conveys the purpose.

Reply:
Agreed with you. Updating these as well. Thanks!